server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/RulesPage.java

```diff
@@ -21,46 +21,108 @@
 
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.ElementsCollection;
-import com.codeborne.selenide.Selenide;
 import com.codeborne.selenide.SelenideElement;
 import org.openqa.selenium.By;
 
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
 public class RulesPage extends Navigation {
 
   public RulesPage() {
-    Selenide.$(By.cssSelector(".coding-rules")).should(Condition.exist);
+    $(By.cssSelector(".coding-rules")).should(exist);
   }
 
   public int getTotal() {
     // warning - number is localized
-    return Integer.parseInt(Selenide.$("#coding-rules-total").text());
+    return Integer.parseInt($("#coding-rules-total").text());
   }
 
   public ElementsCollection getSelectedFacetItems(String facetName) {
-    SelenideElement facet = Selenide.$(".search-navigator-facet-box[data-property='"+ facetName+"']").shouldBe(Condition.visible);
+    SelenideElement facet = $(".search-navigator-facet-box[data-property='"+ facetName+"']").shouldBe(visible);
     return facet.$$(".js-facet.active");
   }
 
   public RulesPage shouldHaveTotalRules(Integer total) {
-    Selenide.$("#coding-rules-total").shouldHave(Condition.text(total.toString()));
+    $("#coding-rules-total").shouldHave(Condition.text(total.toString()));
+    return this;
+  }
+
+  public RulesPage shouldDisplayRules(String ...ruleNames) {
+    for (String ruleName : ruleNames) {
+      $$(".coding-rule").findBy(text(ruleName)).shouldBe(visible);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal ".coding-rule" 5 times. ([squid:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fit-rules&issues=AWEuHWgir8h_fVHZ4iWv&open=AWEuHWgir8h_fVHZ4iWv)

[ISSUE_KEY]: AWEuHWgir8h_fVHZ4iWv